src/account/accountsSelectors.js

```diff
@@ -126,28 +127,40 @@ export const getActiveAccount = (state: GlobalState): Account => {
 export const getCurrentRealm = (state: GlobalState): URL => getActiveAccount(state).realm;
 
 /**
- * The auth object for the active, logged-in account, or undefined if none.
+ * The auth object for this account, if logged in; else undefined.
  *
- * The "active" account is the one currently foregrounded in the UI, if any.
- * The active account is "logged-in" if we have an API key for it.
+ * The account is "logged in" if we have an API key for it.
  *
  * This is for use in early startup, onboarding, account-switch,
- * authentication flows, or other times where there may be no active account
- * or it may not be logged in.
+ * authentication flows, or other times where the given account may not be
+ * logged in.
  *
  * See:
  *  * `getAuth` for use in the bulk of the app, operating on a logged-in
  *    active account.
  */
-export const tryGetAuth: GlobalSelector<Auth | void> = createSelector(
```

Comment:
```
account: Deprecate combined tryGetAuth, adding tryGetThisAuth
```

There are a few places in jsdocs where we say things like "See `tryGetAuth` for the meaning of…". For now, I think those are still reasonably helpful, because `tryGetAuth` still exists and does basically the thing it did before, and its jsdoc refers to specific places where the reader can find current information. Does that sound right?

Once we remove this `tryGetAuth` and rename `tryGetThisAuth` to `tryGetAuth`, though, I think some of those "See `tryGetAuth`…" comments will be unhelpful as they stand now. There are a few that say things like `See tryGetAuth for the meaning of "active, logged-in"`, and the "active" part won't apply anymore.

So I think maybe the thing to do is just make sure we audit those comments when we remove the current `tryGetAuth` and rename `tryGetThisAuth` to `tryGetAuth`. Does that sound right?

Reply:
Yeah. I do that in my draft of the next PR after this, in fact :slightly_smiling_face: 

The resolution to at least one of those references is that it no longer needs to mention "active", because the selector it's documenting will itself become per-account. So the audit depends in part on what the fate of the selector is, and I end up updating that comment in tandem with marking the selector as per-account.